tox.ini

```diff
@@ -66,7 +65,7 @@ commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/systems.git@{posargs:main}
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/models.git@{posargs:main}
 
-    python -m pytest -rsx --cov-config tests/.coveragerc --cov-report term-missing --cov=. tests/integration
+    LD_LIBRARY_PATH="/opt/tritonserver/backends/pytorch:$LD_LIBRARY_PATH" python -m pytest -rsx --cov-config tests/.coveragerc --cov-report term-missing --cov=. tests/integration
```

Comment:
is it required to add `LD_LIBRARY_PATH` to tox's `passenv`?

Reply:
That's not required  because `LD_LIBRARY_PATH` is one of the few that is passed through by default along with other's like `PATH`.

I've updated this PR to keep the logic in `setenv` now